src/Component/src/Doctrine/Persistence/InMemoryRepository.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Doctrine\Persistence;
+
+use ArrayObject;
+use Pagerfanta\Adapter\ArrayAdapter;
+use Pagerfanta\Pagerfanta;
+use Pagerfanta\PagerfantaInterface;
+use Sylius\Component\Resource\Exception\UnexpectedTypeException;
+use Sylius\Resource\Doctrine\Persistence\Exception\ResourceExistsException;
+use Sylius\Resource\Model\ResourceInterface;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\PropertyAccess\PropertyAccessor;
+use Webmozart\Assert\Assert;
+
+class InMemoryRepository implements RepositoryInterface
+{
+    protected PropertyAccessor $accessor;
+
+    protected \ArrayObject $arrayObject;
```

Comment:
ArrayObject is imported at `use` section, but here specified globally

Reply:
yep, I've removed the "use" section.